baseball/src/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from "react";
+import InterfaceDispatch from "../Interface/InterfaceDispatch";
+
+function Dashboard() {
+  const dispatch = useContext(InterfaceDispatch);
+  const incrementStrikes = () => {
+    dispatch({ type: "increment_strikes" });
+  };
+  const incrementBalls = () => {
+    dispatch({ type: "increment_balls" });
+  };
+  const recordFoul = () => {
+    dispatch({ type: "record_foul" });
+  };
+  const recordHit = () => {
+    dispatch({ type: "record_hit" });
+  };
```

Comment:
It seems like you should be able to have a helper function for this; i.e.:
```javascript
function makeDispatch(camelCaseName) {
  const snakeCaseName = camelCaseName.replace(
    /[A-Z]/g, 
    letter => '_' + letter.toLowerCase()
  );

  return () => dispatch({ type: snakeCaseName });
};
```
and the usage:
```javascript
<button onClick={makeDispatch('incrementStrikes')}>strike</button>
```
(you could also name them the same way, just using the makeDispatch helper)